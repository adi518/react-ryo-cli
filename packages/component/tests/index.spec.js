import HelloWorld from "../src/index.js";

describe("HelloWorld", () => {
  it("renders", () => {
    const cmp = shallow(<HelloWorld />);

    expect(cmp).toMatchSnapshot();
  });
});
