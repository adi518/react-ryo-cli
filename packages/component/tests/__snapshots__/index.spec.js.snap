// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld renders 1`] = `"hello world"`;
